plugins.js

```diff
@@ -13,6 +13,14 @@ var states      = require('./connection').states;
 
 var plugin_paths = [path.join(__dirname, './plugins')];
 if (process.env.HARAKA) { plugin_paths.unshift(path.join(process.env.HARAKA, 'plugins')); }
+// Allow environment customized path to plugins in addition to defaults. Multiple paths separated by semi-colon ';'
+if (process.env.HARAKA_PLUGIN_PATH) {
+    var paths = process.env.HARAKA_PLUGIN_PATH.split(';').map(function(p) {
```

Comment:
Can you change this to `.split(/[:;]/)` instead? On unix paths in ENV are generally colon separated (e.g. PATH).